core/templates/dev/head/services/AlertsService.js

```diff
@@ -117,12 +118,16 @@ oppia.factory('AlertsService', ['$log', function($log) {
     AlertsService.messages = newMessages;
   };
 
-  /**
-   * Adds an info message.
-   * @param {string} message - Info message to display.
-   */
-  AlertsService.addInfoMessage = function(message) {
-    AlertsService.addMessage('info', message);
+    /**
+     * Adds an info message.
+     * @param {string} message - Info message to display.
+     * @param timeout - Timeout for the information toast.
+     */
+  AlertsService.addInfoMessage = function(message, timeout) {
```

Comment:
Wouldn't it be better if you add the same parameter passing for success type?.

Reply:
The issue is referenced to a single popup so i didn't think to change the definition of success message. However a default is given in the Directive

Reply:
Yep! But not passing any argument stores an undefined value (timeout) in the addMessage, every time there is a success message. Moreover, the default which you have in the directive will no longer be required. You could directly use message.timeout .

Reply:
Them it will be an optional argument. Correct @ishucr7

Reply:
Done :+1: